packages/browser/src/transports/base.ts

```diff
@@ -5,15 +5,19 @@ import { PromiseBuffer, SentryError } from '@sentry/utils';
 /** Base Transport class implementation */
 export abstract class BaseTransport implements Transport {
   /**
-   * @inheritDoc
+   * @deprecated
```

Comment:
This depends on whether a single transport will eventually have to deal with multiple endpoints like now, or whether we'll have independent transports, each having a fixed target endpoint URL.

Reply:
> I'm not sure if we should deprecate this attribute just yet, as we are not using envelopes for errors.

The transport now always use the `_api` attribute to fetch the appropriate endpoint. `url` is only accessed in tests and possibly user code (e.g. custom transport).